plugins/Files/js/components/filetransfers.js

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react'
+import { List } from 'immutable'
+import TransferList from './transferlist.js'
+
+const FileTransfers = ({uploads, downloads, actions}) => {
+	const onCloseClick = () => actions.hideFileTransfers()
+	let uploadlist
+	if (uploads.size > 0) {
+		uploadlist = (
+			<div className="downloads">
+				<h3> Uploads </h3>
+				<TransferList transfers={uploads} />
+			</div>
+		)
+	}
+	let downloadlist
```

Comment:
the default value is `undefined`.  Rendering `undefined` has the same effect as rendering `null`, but I agree with using the ternary pattern.  I'm going to separate `uploadlist` and `downloadlist` into their own components and ternary them in `FileTransfers`.